cmd/updater/command/command.go

```diff
@@ -35,6 +35,12 @@ type GlobalParams struct {
 
 	// RepositoriesDir is the path to the directory containing the repositories.
 	RepositoriesDir string
+
+	// RunPath is the path to the directory containing the repositories' run data.
+	RunPath string
+
+	// WatchProcesses toggles on the process watcher.
+	WatchProcesses bool
```

Comment:
I think we should support package retention for all package types, there's no need to make an exception it adds many ifs and increases complexity. This updater just supports a generic way to prevent package from being GC'd using the PID mechanism that we introduce here